Framework/src/TaskRunner.cxx

```diff
@@ -57,6 +57,9 @@ TaskRunner::TaskRunner(const std::string& taskName, const std::string& configura
     mCycleNumber(0),
     mTotalNumberObjectsPublished(0)
 {
+  // Log the version
```

Comment:
Can we put it at the beginnings of `InfrastructureGenerator::generateLocalInfrastructure()` and `InfrastructureGenerator::generateRemoteInfrastructure()` instead? These are usually the first QC functions invoked in the code. So if something breaks before we reach the ctor of `TaskRunner`, then we would still see the version number.

Reply:
excellent idea, I am preparing the change